ui/packages/components/src/DatePicker/timeSchemas.ts

```diff
@@ -13,6 +15,20 @@ function getMinMaxHour(is24Format: boolean) {
   }
 }
 
+export const daysInMonth = (year: number, month: months) => {
```

Comment:
I think you don't need this one, as it seems you are using the date-fns helper instead: https://date-fns.org/v3.6.0/docs/getDaysInMonth

Reply:
I wrote that one before I realized it existed in date-fns. I'll delete.